test/python/test_docker_pack.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/python3
+
+import pytest
+import os
+import docker
+import re
+import requests
+import time
+import subprocess
+import tarfile
+
+from utils import project_name
+from utils import basepath
+from utils import tarantool_version
+from utils import tarantool_enterprise_is_used
+from utils import recursive_listdir
+from utils import assert_dir_contents
+from utils import assert_filemodes
+
+
+# #######
+# Helpers
+# #######
+def find_image(docker_client, project_name):
+    for image in docker_client.images.list():
+        for t in image.tags:
+            if t.startswith(project_name):
+                return t
+
+
+def run_command_on_image(docker_client, image_name, command):
+    command = '/bin/bash -c "{}"'.format(command.replace('"', '\\"'))
+    output = docker_client.containers.run(
+        image_name,
+        command,
+        remove=True
+    )
+    return output.decode("utf-8").strip()
+
+
+def wait_for_container_start(container, timeout=10):
```

Comment:
I think it's useful to have in project "wait_for" helper.

`wait_for(function, args, timeout)`